tests/test_scheduler.py

```diff
@@ -1558,3 +1558,31 @@ def test_scheduler_qualify_names():
     assert item.qualify_names(item.children) == (
         ('#routine', 'yet_another_mod#routine', 'other_mod#routine', 'more_mod#routine'),
     )
+
+
+@pytest.mark.parametrize('frontend', available_frontends())
+def test_scheduler_inline_call(here, config, frontend):
+    """
+    Test that inline function calls declared via an explicit interface are added as dependencies.
+    """
+
+    my_config = config.copy()
+    my_config['routine'] = [
+        {
+            'name': 'driver',
+            'role': 'driver'
+        }
+    ]
+
+    scheduler = Scheduler(paths=here/'sources/projInlineCalls', config=my_config, frontend=frontend)
+    print(scheduler.items)
```

Comment:
Debug leftover?

Reply:
Ah yes that is a debug print, I'll remove that and push an update. And are you referring to B's spot that `update_result_var` won't work for a function with an explicitly declared result name? If so, I've fixed that one.

Reply:
Yes, sorry had not understood the fix, but happy now!